tests/functional/serps.php

```diff
@@ -4,35 +4,146 @@
+           'phrase' => 'אתה רוצה כוס התה או כוס קפה או כוס שתייה קלה?',
+		   'language' => 'he',
+			)
+        ),
+				"google global" => array(
+    "request" => array(
+      'region'=>'global',
+          'search_engine' => 'bing',
+           'phrase' => 'best place to take selfie in London',
+		   'language' => 'en',
+			)
+        ),	
+					"google global" => array(
+    "request" => array(
+      'region'=>'global',
+          'search_engine' => 'yahoo',
+           'phrase' => 'best place to take selfie in London',
+		   'language' => 'en',
+			)
+        ),	
+		"google global" => array(
+    "request" => array(
+      'region'=>'global',
+          'search_engine' => 'yandex',
+           'phrase' => 'best place to take selfie in London',
+		   'language' => 'en',
+			)
+        ),
+			"google global" => array(
+    "request" => array(
+      'region'=>'hu',
+          'search_engine' => 'google',
+           'phrase' => 'kérsz egy csésze tea vagy egy csésze kávé vagy egy pohár üdítő?',
+		   'language' => 'hu',
+			)
+        ),
+		"google global" => array(
+    "request" => array(
+      'region'=>'pt',
+          'search_engine' => 'google',
+           'phrase' => 'você quer uma xícara de chá ou uma xícara de café ou um copo de refrigerante?',
+		   'language' => 'pt-PT',
+			)
+        ),
+		"google global" => array(
+    "request" => array(
+      'region'=>'kr',
+          'search_engine' => 'google',
+           'phrase' => '당신은 차 한 잔 또는 커피 한 잔 또는 소프트 드링크 한 잔할까요?',
+		   'language' => 'ko',
+			)
+        ),
+		"google global" => array(
+    "request" => array(
+      'region'=>'in',
+          'search_engine' => 'google',
+           'phrase' => 'আপনি এক কাপ চা বা কফি এক কাপ বা নরম পানীয় একটি কাপ চান?',
+		   'language' => 'bn',
+			)
+        ),
 );
 
 ##################################################################################
 #   Tests that should return an api error message
 ##################################################################################
 $errorTests = array(
-    "invalid language" => array(
-        "request" => array(
-            'region'=>'global',
-            'search_engine' => 'google',
-            'phrase' => 'abc',
+   "invalid language" => array(
+      "request" => array(
+           'region'=>'global',
+           'search_engine' => 'google',
+           'phrase' => 'abc',
             'universal' => 0,
-            'language' => 'asd'
+            'language' => 'asd',
         )
     ),
 );
 
+##################################################################################
+#   Tests that should return an api error message about max results being over 500
+##################################################################################
+$errormaxresults = array(
+    
+);
+
 // Uncommoment the following lines and add the proper values
-// define('API_KEY', 'a');
-// define('API_SECRET', 'b');
-// define('SALT', 'c');
+ define('API_KEY', 'We4vJk3fWvm-2SPq');
```

Comment:
this is an huge security risk,

please remove credentials, maybe better to have them in settings.php file as the file is ignored